src/main/java/com/coherentsolutions/training/aqa/java/zhavrid/util/ChromeDownload.java

```diff
@@ -0,0 +1,16 @@
+package com.coherentsolutions.training.aqa.java.zhavrid.util;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class ChromeDownload {
+    public static void chromeDownload() {
+        String downloadPath = "C:\\Downloads";
```

Comment:
I'm a Mac user. how am I supposed to get anything from this folder?
let's update the path to something inside target dir